incubator/hnc/docs/user-guide/quickstart.md

```diff
@@ -654,3 +655,53 @@ the prefix "Crit," for "Critical") indicates that HNC is no longer updating the
 objects in this namespace. Instead, it's waiting for an admin to come fix the
 problems before it resumes creating or deleting objects. Conditions like these
 also show up in the HNC logs, and in its [metrics](how-to.md#admin-metrics).
+
+<a name="exceptions"/>
+
+## Keeping objects out of certain namespaces
+
+_Demonstrates: exceptions._
+
+Now let’s say your acme-org has a secret that you originally wanted to share with all the teams. We created this secret as follows:
```

Comment:
Format `acme-org` with backticks? Also, maybe format `Secret` as well to be clear that you're talking about the K8s object, and not some kind of horrible information :D